pom.xml

```diff
@@ -41,6 +41,7 @@
         <module>geomesa-security</module>
         <module>geomesa-spark</module>
         <module>geomesa-stream</module>
+        <module>geomesa-test</module>
```

Comment:
Additionally there was talk about moving these test to jars later to give us better test validation.